<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDialog</class>
 <widget class="QWidget" name="TabDialog">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_1">
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="closePB">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="verticalSpacing">
        <number>14</number>
       </property>
       <property name="leftMargin">
        <number>14</number>
       </property>
       <property name="topMargin">
        <number>14</number>
       </property>
       <property name="rightMargin">
        <number>14</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ticketLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="3" colspan="3">
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>Customer Information</string>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Phone #:</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="phoneLineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit">
            <property name="minimumSize">
             <size>
              <width>225</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Order Time:</string>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <spacer name="verticalSpacer_1">
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>95</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Order Ticket #:</string>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Order Date:</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QDateEdit" name="dateEdit">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Build Your Own Sandwich</string>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="horizontalSpacing">
           <number>14</number>
          </property>
          <property name="leftMargin">
           <number>14</number>
          </property>
          <property name="rightMargin">
           <number>14</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="avocado_CB">
            <property name="text">
             <string>Avocado</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="siracha_CB">
            <property name="text">
             <string>Sriracha Sauce</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="mustard_CB">
            <property name="text">
             <string>Mustard</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="jack_CB">
            <property name="text">
             <string>Jack</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chicken_CB">
            <property name="text">
             <string>Roast Chicken</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cornBeef_CB">
            <property name="text">
             <string>Corned Beef</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="lettuce_CB">
            <property name="text">
             <string>Lettuce</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="tomatoes_CB">
            <property name="text">
             <string>Tomatoes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="mayo_CB">
            <property name="text">
             <string>Mayonnaise</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="provolone_CB">
            <property name="text">
             <string>Provolone</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QRadioButton" name="ryeBread_RB">
            <property name="text">
             <string>Rye</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="onions_CB">
            <property name="text">
             <string>Grilled Onions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="bellPeppers_CB">
            <property name="text">
             <string>Bell Peppers</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="veggies_CB">
            <property name="text">
             <string>Grilled Vegetables</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="pastrami_CB">
            <property name="text">
             <string>Pastrami</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="cheddar_CB">
            <property name="text">
             <string>Cheddar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="turkey_CB">
            <property name="text">
             <string>Roast Turkey</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="pickles_CB">
            <property name="text">
             <string>Pickles</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="swiss_CB">
            <property name="text">
             <string>Swiss</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QRadioButton" name="dutchCrunch_RB">
            <property name="text">
             <string>Dutch Crunch</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QRadioButton" name="sourDough_RB">
            <property name="text">
             <string>Sourdough</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QRadioButton" name="wheat_RB">
            <property name="text">
             <string>Wheat</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="horizontalSpacing">
          <number>30</number>
         </property>
         <item row="0" column="3">
          <widget class="QRadioButton" name="salt_RB">
           <property name="text">
            <string>The Salt</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="kitchenSink_RB">
           <property name="text">
            <string>The Kitchen Sink</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="poultryS_RB">
           <property name="text">
            <string>The Poultry Seasoning</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="juniper_RB">
           <property name="text">
            <string>The Juniper</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="rosemary_RB">
           <property name="text">
            <string>The Rosemary</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="coriander_RB">
           <property name="text">
            <string>The Coriander</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_6">
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tab 3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>14</number>
       </property>
       <property name="topMargin">
        <number>14</number>
       </property>
       <property name="rightMargin">
        <number>14</number>
       </property>
       <item row="0" column="4">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Desserts</string>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_38">
            <property name="text">
             <string>Chocolate Chip Cookie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_39">
            <property name="text">
             <string>Lemon Bar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_40">
            <property name="text">
             <string>Peanut Butter Cookie</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Salads</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_35">
            <property name="text">
             <string>Carrot Salad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_36">
            <property name="text">
             <string>Pasta Salad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_37">
            <property name="text">
             <string>Potato Salad</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_9">
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_8">
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Chips</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_31">
            <property name="text">
             <string>BBQ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_32">
            <property name="text">
             <string>Ranch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_33">
            <property name="text">
             <string>Salt &amp;&amp; Pepper</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_34">
            <property name="text">
             <string>Sweet Onion</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="verticalSpacer_4">
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ticketLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>cornBeef_CB</tabstop>
  <tabstop>pastrami_CB</tabstop>
  <tabstop>chicken_CB</tabstop>
  <tabstop>turkey_CB</tabstop>
  <tabstop>veggies_CB</tabstop>
  <tabstop>avocado_CB</tabstop>
  <tabstop>bellPeppers_CB</tabstop>
  <tabstop>onions_CB</tabstop>
  <tabstop>tomatoes_CB</tabstop>
  <tabstop>lettuce_CB</tabstop>
  <tabstop>mayo_CB</tabstop>
  <tabstop>mustard_CB</tabstop>
  <tabstop>pickles_CB</tabstop>
  <tabstop>siracha_CB</tabstop>
  <tabstop>cheddar_CB</tabstop>
  <tabstop>jack_CB</tabstop>
  <tabstop>provolone_CB</tabstop>
  <tabstop>swiss_CB</tabstop>
  <tabstop>dutchCrunch_RB</tabstop>
  <tabstop>ryeBread_RB</tabstop>
  <tabstop>sourDough_RB</tabstop>
  <tabstop>wheat_RB</tabstop>
  <tabstop>coriander_RB</tabstop>
  <tabstop>juniper_RB</tabstop>
  <tabstop>poultryS_RB</tabstop>
  <tabstop>rosemary_RB</tabstop>
  <tabstop>salt_RB</tabstop>
  <tabstop>kitchenSink_RB</tabstop>
  <tabstop>checkBox_31</tabstop>
  <tabstop>checkBox_32</tabstop>
  <tabstop>checkBox_33</tabstop>
  <tabstop>checkBox_34</tabstop>
  <tabstop>checkBox_35</tabstop>
  <tabstop>checkBox_36</tabstop>
  <tabstop>checkBox_37</tabstop>
  <tabstop>checkBox_38</tabstop>
  <tabstop>checkBox_39</tabstop>
  <tabstop>checkBox_40</tabstop>
  <tabstop>closePB</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
